setup.py

```diff
@@ -1,9 +1,23 @@
 #!/usr/bin/env python
 from setuptools import find_packages, setup
 import versioneer
+import sys
 
 README = open('README.rst', 'r').read()
 
+install_requires = [
+        'configparser; python_version<"3.0"',
+        'future>=0.14.0; python_version<"3"',
+        'backports.functools_lru_cache; python_version<"3.2"',
+        'jedi>=0.17.0,<0.18.0',
+        'python-jsonrpc-server>=0.3.2',
```

Comment:
```suggestion
        'python-jsonrpc-server>=0.4.0',
```

@steff456, with this change we'll bring compatibility between this project and `python-jsonrpc-server` again.